README.md

```diff
@@ -89,3 +73,21 @@ d       1.*.5                                       error   unsupported format
     - run `git log` to identify # of commits 
     - add commit count to the patch version value from the latest tag
     - viola! we got the version to use!
+
+## Similar plugins
+
+There are other plugins out there that are similar:
+
+1. Below 2 plugins are great but they (mostly) require to push a tag to make a release.
+Our plugin can release every change. 
+ - https://github.com/ajoberstar/reckon
+ - https://github.com/allegro/axion-release-plugin
+ - https://github.com/cinnober/semver-git
```

Comment:
Please also add https://github.com/nemerosa/versioning here. It's very good plugin

Reply:
Can you add this plugin with a little narrative? (perhaps 1 sentence describing it)? Thanks!